tensorrt_llm/llmapi/trtllm-llmapi-launch

```diff
@@ -19,7 +19,8 @@ function mpi_world_size {
     elif [ -n "$OMPI_COMM_WORLD_SIZE" ]; then
         echo "$OMPI_COMM_WORLD_SIZE"
     else
-        echo "1"
+        echo "Error: Unable to determine MPI world size. Neither SLURM_NTASKS nor OMPI_COMM_WORLD_SIZE is set." >&2
+        exit 1
```

Comment:
So are we implying here that we only support `trtllm-llmapi-launch` on Slurm clusters or with OpenMPI? Do we know if customers use it in different environments?